Admin/AbstractAdmin.php

```diff
@@ -422,8 +422,6 @@
      */
     protected $securityInformation = array();
 
-    protected $cacheIsGranted = array();
```

Comment:
Oh right, but neither would your example work then, I guess… BTW, what do you mean by "noop" here?